spec/azure/virtual_network.go

```diff
@@ -0,0 +1,7 @@
+package azure
+
+type VirtualNetwork struct {
+	CIDR             string `json:"cidr" yaml:"cidr"`
```

Comment:
Could it be possible to update the json and yaml tags to match the ones used in the [ARM templates](https://github.com/giantswarm/azure-operator/pull/16/files#diff-19eeeb177c7950c3a8ab163c3f04bfb0R15)?

I can also update my own PR to match your tags without any issues :)

Reply:
We need to use this format because Kubernetes uses it for serialising and deserialising the TPOs.

I like having the ARM templates matching for consistency. So if you're OK to change that would be good. But if there are naming conventions for the ARM templates maybe we should follow those?